src/core/stdcpp/array.d

```diff
@@ -0,0 +1,91 @@
+﻿/**
+ * D header file for interaction with C++ std::array.
+ *
+ * Copyright: Manu Evans 2014 - 2018.
```

Comment:
That'd be problematic I think

Reply:
True.. what's the proper copyright assignment?
I just copied what I found in neighbouring code.

Reply:
To the "D Language Foundation"